docs/operation/cluster_monitoring/prometheus.rst

```diff
@@ -28,3 +28,20 @@ To customize predefined alert rules, refer to
 .. csv-table:: Default Prometheus Alerting rules
    :file: ../../../tools/rule_extractor/alerting_rules.csv
    :header: "Name", "Severity", "Description"
+
+Snapshot Prometheus Database
+----------------------------
+
+Snapshotting Prometheus database can be useful for backup and restore,
+but also to export the platform metrics in order to investigate, for
+example, a performance issue.
+
+To snapshot the database, you must first
+:ref:`enable the Prometheus admin API<csc-enable-prometheus-admin-api>`.
+
+You can then generate the snapshot using
+:ref:`sosreport utility<sosreport-generate-report>` with the following options:
+
+.. code-block:: shell
+
+   sosreport --batch -o metalk8s -kprometheus-snapshot=True
```

Comment:
OK, but where is the snapshot generated? Any concern about size?

Reply:
Added an explanation and a warning.